dev-tools/xtask/src/illumos.rs

```diff
@@ -0,0 +1,144 @@
+use anyhow::{bail, Context, Result};
```

Comment:
MPL files require a license header up top -- could you add one here?

Reply:
Fixed in [fa5b905](https://github.com/oxidecomputer/omicron/pull/5259/commits/fa5b905ef38a8a5f50130329967961514346ec67)